src/components/PathwayExercise/index.js

```diff
@@ -125,6 +158,31 @@ function PathwayExercise() {
         console.log("error");
       });
   }, []);
+  useEffect(() => {
+    axios({
+      method: METHODS.GET,
+      url: `${process.env.REACT_APP_MERAKI_URL}progressTracking/learningTrackStatus`,
+      headers: {
+        "version-code": versionCode,
+        accept: "application/json",
+        Authorization: user.data?.token || "",
+      },
+    }).then((res) => {
+      const data = res.data.data;
+      const filteredData = data.filter((item) => {
+        if (
+          params.pathwayId == item.pathway_id &&
+          params.courseId == item.course_id
+        ) {
+          return item;
+        }
+      });
```

Comment:
You'll always want the function you pass to `filter` to return something to indicate whether to keep it (truthy value, but generally just `true`) or not (falsey value).  This implicitly does that because without an explicitly given return value, the function returns the falsey `undefined`.  But it should be made explicit.
```suggestion
      const filteredData = data.filter((item) =>
          params.pathwayId == item.pathway_id &&
          params.courseId == item.course_id
      );
```